code/game/jobs/job/religions.dm

```diff
@@ -563,10 +570,90 @@
 	female_adept = "Vampire"
 	keys = list("vampire", "vamp", "blood","dracula")
 
+/datum/religion/vampirism/misc(var/mob/living/carbon/human/H)
+	H.equip_or_collect(new /obj/item/clothing/suit/storage/draculacoat(H), slot_wear_suit)//What could possibly go wrong?
+
 /datum/religion/vox
 	name = "Voxophilia"
 	deity_name = "The Vox"
 	bible_name = "Handbook to the Aves Class" //AKA birds
 	male_adept = "Vox Enthusiast" //And that's terrible
 	female_adept = "Vox Enthusiast"
 	keys = list("vox", "raiders", "raid", "bird", "birb")
+
+/datum/religion/bleb
+	name = "Blob Worship"
+	deity_name = "Blob Overmind"
+	bible_name = "A Guide To Biohazard Alerts"
+	male_adept = "Blob Core"
+	female_adept = "Blob Core"
+	keys = list("blob", "bleb", "biohazard")
+
+/datum/religion/clown
+	name = "Clownism"
+	deity_name = "Honkmother"
+	bible_name =  "Honkmothers Coloring Book"
+	male_adept = "Co-Clown"
+	female_adept = "Co-Clown"
+	keys = list("honk", "clown", "honkmother")
+
+/datum/religion/clown/misc(var/mob/living/carbon/human/H)
+	H.equip_or_collect(new /obj/item/clothing/mask/gas/clown_hat(H), slot_wear_mask)
+
+/datum/religion/mime
```

Comment:
This should give the chaplain the silence mutation, if that's within scope

Reply:
@MadmanMartian That would make them strictly speaking mimes with a bible, though

Reply:
It'd actually make them strictly silent mimes.